test/unit/org/apache/cassandra/index/sai/cql/types/QuerySet.java

```diff
@@ -178,7 +155,7 @@ protected Object[][] getExpectedRows(Object value, Object[][] allRows)
             List<Object[]> expected = new ArrayList<>();
             for (Object[] row : allRows)
             {
-                if (((Collection)row[2]).contains(value))
+                if (((Collection<?>)row[2]).contains(value))
```

Comment:
nit: The magic number `2` makes me a little worried. Maybe a named constant would be useful?